tests/test_rdscluster.py

```diff
@@ -347,6 +347,36 @@ def test_start(self):
             DBClusterIdentifier='mytest').get('DBClusters')[0]
         self.assertEqual(cluster['Status'], 'starting')
 
+    def test_modify_rds_cluster(self):
+        session_factory = self.replay_flight_data("test_modify_rds_cluster")
+        p = self.load_policy(
+            {
+                "name": "modify-db-cluster",
+                "resource": "rds-cluster",
+                "filters": [{"DBClusterIdentifier": "c7n-test-cluster"}],
```

Comment:
this should be filtering for delete protection as turned on, else this whole test could be a no-op.